raster_api/runtime/setup.py

```diff
@@ -9,6 +9,7 @@
     "titiler.pgstac==0.1.0.a9",
     "titiler.application>=0.5,<0.6",
     "starlette-cramjam>=0.1.0,<0.2",
+    "fastapi==0.93.0",
```

Comment:
Comparing logs between 🟢 and 🔴 I saw that the only differences were the FastAPI and Starlette versions. This PR fix FastAPI to the latest `working` version. 

I'll check in titiler-pgstac if something is broken with FastAPI>=0.94